test/helpers/apply-transform.js

```diff
@@ -29,6 +27,7 @@ export function dedent(raw, ...values) {
 
 // Call the transform function with the appropriate arguments
 export default function applyTransform(module, options = {}, source, path = 'test.js') {
+  let jscodeshift = require('../../.codeshift/js/node_modules/jscodeshift');
```

Comment:
If this path were shared, we could do something like `require(codeshift.js.path)` here